src/contextmenu.js

```diff
@@ -150,8 +152,11 @@ export class ContextMenu {
         clampTimeout = timeout(() => clampSubMenu(this.core.$root, ev));
       },
       show: (options) => (props, actions) => {
-        let {menu, position} = options;
-        if (position instanceof Event) {
+        let {menu, position, toggle} = options;
+        if (toggle && isActive) {
+          return actions.hide();
```

Comment:
Place the `return;` on its own line.

Reply:
Uhm I thought the action has to return something. Initially it was `actions.hide(); return;` but then the return value from `hide` is lost and `show` returns `undefined`. It worked somehow, so perhaps it was ok?

Reply:
Also I noticed that sometimes you use `actions` (the argument) and sometimes `this.actions` (the member). Which is preferable?

Reply:
Yeah. In hindsight it's totally fine to just leave it there.

> Also I noticed that sometimes you use actions (the argument) and sometimes this.actions (the member). Which is preferable?

The former (argument) is probably a more natural way of doing this.

Reply:
Ok, I made all the actions use the argument and fixed the styling issues.